hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -110,44 +110,9 @@ class DefaultSource extends RelationProvider
     val isBootstrappedTable = metaClient.getTableConfig.getBootstrapBasePath.isPresent
```

Comment:
Let's move these statements (up to line #109) into `DefaultSource.createRelation` to make a cleaner cut (all these vars are used in that method rather than here)